GMC_identification.ipynb

```diff
@@ -0,0 +1,1603 @@
+{
```

Comment:
<p>Major:</p><p>"import matplotlib.pyplot as plt"</p><p>Needed this to run on the version I downloaded</p><p> </p><p>Minor:</p><p>Color-schemes differ between my version and here (if the image coloring and scale is important to you, then cmap, vmin vmax etc need specifying)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>added!</p>